charts/jenkins/jobs/nuxeo-compound-documents/nuxeo-compound-documents.groovy.gotmpl

```diff
@@ -0,0 +1,40 @@
+multibranchPipelineJob('nuxeo/nuxeo-compound-documents') {
+  // Description
+  description('Files that only make sense as a group of interrelated children, possibly at several levels, each of these is modifiable.')
+  // Branch Sources
+  branchSources {
+    branchSource {
+      source {
+        // GitHub
+        github {
+          id('nuxeo/nuxeo-compound-documents')
```

Comment:
compound-documents/nuxeo-compound-documents (we use the same as the upper defined job path)